### DISCUSSION TITLE
Web Component for version HTML

### DISCUSSION THREAD

**Discussion by seb-jean** [tailwindlabs/heroicons#538] [Ideas]
Hi,
Currently, each icon can be imported individually as a React/Vue component

For instance for React:
```js
<BeakerIcon className="h-5 w-5 text-blue-500"/>
```

Or for Vue:
```js
<BeakerIcon class="h-5 w-5 text-blue-500"/>
```

It would be nice to have the same genre for the HTML version.
For instance :

```html
<beaker-icon class="h-5 w-5 text-blue-500"/>
```

Thanks

**Comment by Avelar** [comment]
We would love this feature!

**Comment by marcosvafilho** [comment]
That would be awesome.

I was looking for how to use Heroicons similar to Font Awesome, by adding css classes (i.e. `hi-star`), but native html web components would be waaaaaay better.
